docs/source/contributing/developing.adoc

```diff
@@ -106,9 +106,9 @@ Make sure to use Dep version 0.4.1 or later.
 [[bootstrap-dependencies]]
 === Bootstrapping Dependencies
 
-After a clean checkout or after a `make clean`, there won't be a *_vendor_* directory containing the needed {project} dependencies.
+If you delete *_vendor_* directory which contain the needed {project} dependencies.
```

Comment:
Now that the code repository will contain the vendor directory, this is odd as the first sentence of `Bootstrapping Dependencies`. I would suggest we this sentence after `Adding a new Dependency` with a new title e.g. `recreating vendor directory`.

Reply:
I was not sure about that, but I will do that modification.